### PR TITLE
ADD srv and subsrv headers to data API examples

### LINKED ISSUE #65: Missing headers in CB subscribe to data updates
in http://fiware-iot-stack.readthedocs.io/en/release-v4.1/data_api/index.html#subscribe-to-data-updates there aren't the headers fiware-service, fiware-servicepath and x-auth-token.

It would nice to inform the integrator that these headers are mandatory to susbcribe to data updates from the point of view of the platform.

### PR DESCRIPTION
**Pull request by fgalan**
Fixes #65

### CODE REVIEW

File: docs/data_api.md [hunk 1]
```diff
@@ -12,13 +12,20 @@ Before continuing with the next section, it would be a good idea to have a look
...
 
-Entities that would be relevant to the NiceEating application are of type Restaurant, Client and Rating. For example, when a given user scores a restaurant (e.g. in a scale from 0 to 5, “Client1234” scores “4” for the “LeBistro” restaurant) the smartphone application plays the Context Producer role **creating** a Rating entity by issuing the following HTTP request :
+Entities that would be relevant to the NiceEating application are of type Restaurant, Client and Rating. For example, when a given user scores a restaurant (e.g. in a scale from 0 to 5, “Client1234” scores “4” for the “LeBistro” restaurant) the smartphone application plays the Context Producer role **creating** a Rating entity by issuing the following HTTP request:
 
 ```
 POST <cb_host>:<cb_port>/v2/entities
 Content-Type: application/json
+Fiware-service: smartown
```
Comment (fgalan) [inline docs/data_api.md]: I'm unsing the same sample service that in other places in the doc (e.g. http://fiware-iot-stack.readthedocs.io/en/release-v4.1/multitenancy/index.html) in order to keep documentation consistent. I recommend other contributors do the same.

NTC (just informative)

**Comment by manucarrace** [comment]
I would also include the x-auth-token header for completeness

File: docs/data_api.md [hunk 2]
```diff
@@ -12,13 +12,23 @@ Before continuing with the next section, it would be a good idea to have a look
...
+Fiware-servicepath: /NiceEating
+X-Auth-Token: D3wiv5j7y6oiwo9w4ds5u20Y0bb6pL
```
Comment (fgalan) [inline docs/data_api.md]: Before you ask :), this is a fake token.

NTC (just informative)

**Comment by fgalan** [comment]
Good point. X-Auth-Token added to examples in commit ea278f610ea9abfbdec57ee3304fb06a3173b805

**Comment by manucarrace** [comment]
LGTM

**Comment by fgalan** [comment]
(Good practise: delete branches after merging the branch. Keep repositories neat and tidy ;)

**Pull request merged**